WordPress/Classes/ViewRelated/Stats/Extensions/Double+Stats.swift

```diff
@@ -157,16 +161,28 @@ extension NSNumber {
     func abbreviatedString(forHeroNumber: Bool = false) -> String {
         return self.doubleValue.abbreviatedString(forHeroNumber: forHeroNumber)
     }
+
+    func percentageString() -> String {
+        return self.doubleValue.percentageString()
+    }
 }
 
 extension Float {
     func abbreviatedString(forHeroNumber: Bool = false) -> String {
         return Double(self).abbreviatedString(forHeroNumber: forHeroNumber)
     }
+
+    func percentageString(forHeroNumber: Bool = false) -> String {
```

Comment:
I just noticed that `forHeroNumber` could be removed here.

Reply:
You're right https://github.com/wordpress-mobile/WordPress-iOS/pull/22957